tests/test_relperm.py

```diff
@@ -125,7 +126,7 @@ def test_relperm(tmp_path):
    assert len(relperm_parts_in_model(model, low_sal = False)) == 1
    dataframe1.df_to_text(filepath = tmp_path, filename = 'oil_water_test_table')
    df1_reconstructed = text_to_relperm_dict(os.path.join(tmp_path, 'oil_water_test_table.dat'))['relperm_table1']['df']
-   assert df1.equals(df1_reconstructed)
+   assert_frame_equal(df1, df1_reconstructed)
```

Comment:
This is such a useful function...thanks for sharing Connor. The changes look good.